src/schema/metadata/RepetitionTimeExcitation.yaml

```
@@ -0,0 +1,20 @@
+---
+name: RepetitionTimeExcitation
+description: |
+  The interval, in seconds, between two successive excitations.
+  The DICOM tag that best refers to this parameter is
+  [(0018, 0080)](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0080)).
```

casing typo: tag -> Tag

Also I think this should be rephrased to something like: DICOM tag xxxx, xxxx `Name Of This Thing` best refers to this parameter.

We usually don't link DICOM tags in links, so I don't know why we should do it here. Happy to keep it if there is a good argument, but regardless I think we should use the rephrasing I propose above to be consistent.

---

I think this might be a question for the qMRI folks, and I don't want to pull additional people into this PR so late in the game. Is it cool if we push this to a separate issue?

---

Yes, fine with me to deal with it another time, but my suggestion below would not require qMRI folks input, I think. And it'd align the way we describe DICOM tags in the spec better.

```suggestion
  The interval, in seconds, between two successive excitations.
  [DICOM Tag 0018, 0080](http://dicomlookup.com/lookup.asp?sw=Tnumber&q=(0018,0080)
  best refers to this parameter.
```

---

Ah, that's a great suggestion. My main concern was about dropping the link, since it looks like there are two definitions for that tag, and I'm not sure which we're referring to.